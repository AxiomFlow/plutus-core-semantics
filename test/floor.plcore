(module Foo (imported) (exported () ()) (local () ())
(define result (builtin floor (2.5)))
)
