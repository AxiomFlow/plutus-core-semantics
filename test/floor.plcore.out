<T> <k> 2e+00 </k> </T>
